src/app/shared/group-title/group-title.component.pug

```diff
@@ -0,0 +1,11 @@
+.row.group-border
+    .col-9.pb-24.pt-24.z-1000
+        .row.m-sm-0.pb-3(#titlerow="")
+            h1.title {{ ig.name }}
+        .row.m-sm-0.item-last-updated
+            span {{ ig.total }} items
+            span &nbsp;&nbsp;•&nbsp;&nbsp;
+            span Last updated {{ ig.update_date | date:'MMMM dd, yyyy' }}
+    .col-3.pb-24.pt-24.z-1000
```

Comment:
Rather than using padding/margin helpers, we have a handy ".btn-row" class. One day we'll have styleguide that include all these utilities 😂 
But for now, if you could use:
.btn-row
     .btn
     .btn 
Your spacing should be handled for you